content/hilfe/Handbuch/Mitarbeiterapp2/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Mitarbeiter-App2
+weight: 1
+icon: mobile
+---
+
+-- Funktionen der Mitarbeiterapp - Installation separat
+
+
+
+## Einloggen
+
+Nachdem Sie den Mitarbeiter wie unter 
+[Zugänge &Rechte](/hilfe/handbuch/mitarbeiter/zugaenge-rechte) beschrieben per Email
+eingeladen haben und er sich registriert hat, kann er sich nun in der
+Mitarbeiterapp einloggen. Hierzu ruft er die Mitarbeiterapp unter
+https://perso.pentacode.app auf und gibt seine Email Adresse, sowie das
+selbstgewählte Passwort ein. Worauf man in die **Übersicht** der Mitarbeiterapp
+gelangt.
+
+## Übersicht
+
+In der Übersicht sehen Sie ganz oben ihr aktuelles Profilbild, den Namen ihres
+Unternehmens sowie ihre Stunden, Überstunden und Resturlaub. **Stunden** zeigt
+Ihnen an, wie viele der vertraglich vereinbarten Stunden sie diesen Monat
+bereits abgearbeitet haben. **Überstunden** zeigt Ihnen ihr Pensum an
+Überstunden an, wobei ein **rotes Minus** vor der Zahl für negative und ein
+**grünes Plus** für positive Überstunden steht. **Resturlaub** gibt die
+restlichen Tage gesetzlich geregelten Urlaubs für das laufende Jahr an. Alle
+Angaben werden regelmäßig auf den aktuellen Monat aktualisiert.
+
+{{<figure caption="Dieser Mitarbeiter ist zum Beispiel erst 8 von 85 Stundendiesen Monat beschäftigt gewesen, muss 65 negative Überstunden nachholen und hat unter einem Tag Resturlaub" >}} 
+<img src="übersicht.png" /> 
+{{</figure>}}
+
+### Bild ändern
+
+So ändern Sie ihr Bild:
+1. Klicken Sie auf den {{<icon "camera">}}-**Button** neben ihrem aktuellen
+   Profilbild.
+2. Klicken Sie im aufpoppenden Dialogfenster auf **zulassen**, wenn Sie gefragt
+   werden ob Sie Pentacode Zugriff auf ihre Kamera gewähren wollen.
+3. Es öffnet sich die Ansicht der Innenkamera - nun können Sie ein neues Bild
+   machen oder per Klick auf {{<icon "image">}} ein vorhandenes Bild von ihrem
+   Gerät verwenden. 
+
+## Zeiterfassung
+
+Unter dem Reiter Zeiterfassung sehen Sie, ob an diesem Tag eine Schicht für Sie
+ansteht und können für diese einstempeln, sofern sie nicht das System der
+digitalen [Stempeluhr](/hilfe/handbuch/stempeluhr) nutzen. Sollten das
+**Stempeln via. App** erlaubt sein, nutzen Sie diese Funktion wie die Stempeluhr
+(Erklärung [hier](/hilfe/handbuch/stempeluhr)).
+
+### Spontane Schicht
+
+Sollten Sie sich mit ihrem Vorgesetzten darauf geeinigt haben auch spontane
+Schichten zu übernehmen, können Sie diese auch unter Zeiterfassung starten.
+Wählen Sie erst eine **Position** in der Sie die spontane Schicht antreten und
+Klicken anschließend auf **Spontane Schicht** und **Bestätigen**.
```

Comment:
Der ganze Bereich "Zeiterfassung" kann noch deutlich ausführlicher behandelt werden:

- Einstempeln
    - Mit QR-Code
    - Ohne QR-Code
    - Spontane Schicht
- Ausstempeln
    - Mit QR-Code
    - Ohne QR-Code
    - Beschreibung Quickinfo die beim Ausstempeln angezeigt wird
    - Was passiert wenn vergessen wird, auszustempeln?
- Pausen erfassen
    - In die Pause Ausstempeln
    - Wieder einstempeln
 
Wegen der Länge brauchst du dir keine Sorgen machen, dafür gibt es ja das Inhaltsverzeichnis auf der linken Seite. Und falls es doch zu viel für einen Artikel wird können wir uns überlegen für die einzelnen Bereiche (Zeiterfassung, Dienstplan, Abwesenheiten etc.) eigene Artikel machen.

Reply:
Separarte Zielgruppe, deswegen inhaltmäßige Überschneidungen